packages/react/src/types/component-props.ts

```diff
@@ -254,7 +298,15 @@ export interface ReactFlowProps<NodeType extends Node = Node, EdgeType extends E
   /**
    * Custom edge types to be available in a flow.
    *
-   * React Flow matches an edge's type to a component in the edgeTypes object.
+   * React Flow matches an edge's type to a component in the `edgeTypes` object.
+   * @TODO check if @default is correct
+   * @default {
+   *   default: BezierEdge,
+   *   straight: StraightEdge,
+   *   step: StepEdge,
+   *   smoothstep: SmoothStepEdge,
+   *   simplebezier: SimpleBezier
+   * }
```

Comment:
```suggestion
   * @default {
   *   default: BezierEdge,
   *   straight: StraightEdge,
   *   step: StepEdge,
   *   smoothstep: SmoothStepEdge,
   *   simplebezier: SimpleBezier
   * }
```
I copied `@default` from the reactflow API reference. Can you agree on whether it is correct?